src/main/java/org/example/Person.java

```diff
@@ -34,7 +34,7 @@ public void addSibling(Person sibling){
      siblings.add(sibling);
     }
 
-    public List<Person> addGrandChildren(){
+    public List<Person> addGrandChildren(Person grandChildren1){
```

Comment:
Volgens mij gaf je het ook in de les aan dat je add.grandchildren methode niet lukte. Dit is volgens mij omdat je de achter klein kinderen oproept. Je neemt namelijk grandchild.getchildren, dus je gaat nog een laagje dieper. Herkenbaar, want ik had dit dus ook omdat dit in het huiswerk klasje was gezegd. Wat ik heb gedaan: public 

List<Person> getGrandChildren() {
        ArrayList<Person> grandChildren = new ArrayList<>();
        for (Person child : getChildren()) {
            for (Person grandchild : child.getChildren()) {
                grandChildren.add(grandchild);
            }
            return grandChildren;